spec/event_builder_spec.rb

```diff
@@ -52,7 +48,7 @@
 
     expect(@event_builder).to receive(:create_impression_event)
                           .with('test_experiment', 'test_user')
-                          .and_return(Optimizely::Event.new(expected_params))
+                          .and_return(Optimizely::Event.new(:get, '', expected_params))
```

Comment:
for completeness, we should also have a test case to check that the event is built with the correct method and url